exercises/python/trivia.py

```diff
@@ -119,38 +133,44 @@ def was_correctly_answered(self):
 
             print("Answer was corrent!!!!")
             self.purses[self.current_player] += 1
-            print(self.players[self.current_player] + \
+            print(self.players.playerList[self.current_player] + \
                 ' now has ' + \
                 str(self.purses[self.current_player]) + \
                 ' Gold Coins.')
 
             winner = self._did_player_win()
             self.current_player += 1
-            if self.current_player == len(self.players): self.current_player = 0
+            if self.current_player == len(self.players.playerList): self.current_player = 0
 
             return winner
 
     def wrong_answer(self):
         print('Question was incorrectly answered')
-        print(self.players[self.current_player] + " was sent to the penalty box")
-        self.in_penalty_box[self.current_player] = True
+        print(self.players.playerList[self.current_player] + " was sent to the penalty box")
+        self.put_player_in_penalty_box(self.current_player)
 
         self.current_player += 1
-        if self.current_player == len(self.players): self.current_player = 0
+        if self.current_player == len(self.players.playerList): self.current_player = 0
         return True
 
+    def put_player_in_penalty_box(self, player):
```

Comment:
clear names - can clearly see the difference between is in penalty box and put player in penalty box